README.md

```diff
@@ -56,6 +56,12 @@ The root account is derived from the network passphrase and if the network passp
 
 *Note*: The standalone network in this container is not suitable for any production use as it has a fixed root account. Any private network intended for production use would also required a unique network passphrase.
 
+In order to get started quickly, you can deploy the docker image to a DigitalOcean server by clicking the button below. It will create a container in *ephemeral mode* on the *standalone network* that can be connected to. Note: you will need to create a DigitalOcean account if you don't have one.
+
+[![Deploy to DO](https://www.deploytodo.com/do-btn-blue.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/stellar/docker-stellar-core-horizon/tree/acharb-quickdeploy)
+
+*disclaimer*: this creates a new private network that DigitalOcean makes public. Do not put sensitive information on the network that you would not someone else to know.
```

Comment:
❗ I think it's important to note that the root account is known and so the native asset on the network won't be secure. Also, let's use appropriate capitalization consistent with the rest of the doc.
```suggestion
*Disclaimer*: The DigitalOcean server is publicly accessible on the Internet. Do not put sensitive information on the network that you would not someone else to know. Anyone with access to the network will be able to use the root account above unless.
```

Reply:
🤔 It would be better practice if any publicly accessible deployment used a separate network passphrase, and so it would be a good idea for us to set a good example.

We could make the DigitalOcean deployment use a random network passphrase. It would be relatively easy to do this by adding a `--randomize-network-passphrase` option to the `start` script that is only valid for use with the `--standalone` option. When that option is set we inject some random numbers/letters into the standalone network passphrase string that is set here:
https://github.com/stellar/docker-stellar-core-horizon/blob/5d3db26f80cb7b2ab0a5bf61fabbaa00b2b786c4/start#L109-L110 

It would make the publicly deployed instance a little more secure since txs signed for one deployment are not usable on another deployment. It doesn't solve the root account issue as the root account is still determinable which is no different to standalone.

Thoughts?

Reply:
i'm not opposed to adding that feature. But if someone gets the URL of the deployment they'd be able to get the network passphrase pretty easily. So maybe adding a random network passphrase gives a false sense of security? So the only way to ensure the network isn't access by others is keeping the url secure, since DO doesn't have App whitelisting afaik

I don't think the above is that much of a problem though for the use case of someone wanting to spin up a network to play around in

Reply:
You're correct, this doesn't make the root account anymore secret, and in both the standalone and standalone-with-random case the root account can be known, and can be secured.

There are two issues at play here which I conflated. 

The way to secure the root account is to generate another account whose key is unknown to anyone but the holder, and to transfer all the assets to that new key, then to lock the root account.

Making the network passphrase on the deployment unique ensures that a transaction signed for this network is not valid on another network, which is a best practice we should encourage, and is mentioned here:

https://github.com/stellar/docker-stellar-core-horizon/blob/92e82b2b65a49f7831543f3409ad35da1894e33b/README.md#L57

I think the effort to add a random hash to the passphrase should be pretty small, so I think we could do that?

Maybe @MonsieurNicolas can provide some perspective on if this is worth it or not.

Reply:
> Making the network passphrase on the deployment unique ensures that a transaction signed for this network is not valid on another network, which is a best practice we should encourage

gotchya I hadn't thought about that. Added the flag here

https://github.com/stellar/docker-stellar-core-horizon/pull/276/commits/1f8f39c85c58610f3415116697b933a1d04d137f

note: I used $RANDOMIZE for the random string, but if there's a better method lmk